src/test/java/organice/logic/commands/CommandTestUtil.java

```diff
@@ -40,20 +45,38 @@
     public static final String VALID_NRIC_DOCTOR_AMY = "S1111111A";
     public static final String VALID_NRIC_DONOR_JOHN = "T1312123P";
     public static final String VALID_NRIC_PATIENT_BOB = "G2222222B";
-    public static final String VALID_NRIC_PATIENT_IRENE = "S1111111A";
```

Comment:
Basically its better for each person to have different fields value so that when we do testing, usually the `same attributes but different age` we can be sure the values are different. Previously the value of `doctor_amy` nric is the same as `patient_irene` and if we want to test or e.g. `isSamePerson`, `doctor_amy` is same person as `irene` which is not the case.

Reply:
Aiight